### ISSUE TITLE
Failed to capture traffic

### ISSUE THREAD

**Issue by Jondoe398** [nmap/npcap#733]
I installed npcap on a Windows tablet device. The network adapter of this device is Realtek PCIe GbE Family, and the network card driver is rt86win7.sys. Npcap failed to capture the network traffic of this adapter and also failed to mount

**Comment by gvanem** [comment]
I use a similar RealTek PCIe GbE driver w/o problems on Win-10, 64-bit (driver version 10.54.1111.2021).

But you seems to be using Win-7, 32-bit (?). Not sure if NPcap supports that.
Check the output of `systeminfo | find "System Type:"`

**Comment by Jondoe398** [comment]
> I use a similar RealTek PCIe GbE driver w/o problems on Win-10, 64-bit (driver version 10.54.1111.2021).
> 
> But you seems to be using Win-7, 32-bit (?). Not sure if NPcap supports that. Check the output of `systeminfo | find "System Type:"`
> 
> _Edit_: See #536 for some details on this.

Yes, this is an x86 device. The system information shows it as an X86-based PC.
